.circleci/config.yml

```diff
@@ -118,7 +116,7 @@ commands:
             apt-get update && apt-get install -y nodejs
       - run:
           name: Install serverless CLI
-          command: npm i -g serverless@^3
+          command: npm i -g serverless
```

Comment:
Seems alright, though, I'm unsure whether this will work out.

I think Ryan said we pivoted to npx way of doing it because there were some nuance issues with installing serverless plugin packages.

But then again, I haven't seen that issue myself, so we can give it a go this way.